src/utils/utils.ts

```diff
@@ -240,3 +240,7 @@ export function assert(x: any, ...args: any): asserts x {
     console.assert(x, ...args)
     if (!x) throw new Error('Assert failed!')
 }
+
+export function checkInputLengthExceed(name: string) {
+    return Array.from(name).length > WALLET_OR_PERSONA_NAME_MAX_LEN
```

Comment:
```suggestion
    return name && Array.from(name).length > WALLET_OR_PERSONA_NAME_MAX_LEN
```

Reply:
done

Reply:
Why add this here?

Reply:
```
disabled={!name || checkInputLengthExceed(name)}
```

Reply:
🤔I don't quite follow this, type of `name` is always string, maybe we can change this to：

```
disabled={name.length === 0 || checkInputLengthExceed(name)}
```

Reply:
OK

Reply:
```suggestion
    return Array.from(name).length >= WALLET_OR_PERSONA_NAME_MAX_LEN
```

i think, so `length: WALLET_OR_PERSONA_NAME_MAX_LEN - 1,` is doesn't need.

Reply:
done